openeo/extra/job_management.py

```diff
@@ -458,3 +422,60 @@ def ignore_connection_errors(context: Optional[str] = None):
         _log.warning(f"Ignoring connection error (context {context or 'n/a'}): {e}")
         # Back off a bit
         time.sleep(5)
+
+
+class JobTrackerStorage:
+    # TODO: add support to store as Parquet file as well as CSV.
+    # TODO: add support to store to both PosixPath and URL.
+    """
+    Helper to manage the storage of batch job metadata.
+    """
+
+    def __init__(self):
+        pass
+
+    def normalize_df(self, df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
I'm not sure the dataframe normalization is an aspect of the storage. It's not that a different storage class will require different columns/defaults. I think the normalization can stay part of the job manager